apps/extension/client/package.json

```diff
@@ -81,6 +81,20 @@
           "type": "boolean",
           "default": true,
           "description": "Enable nesting the generated typegen files."
+        },
+        "xstate.sendChanges": {
+          "type": "string",
+          "enum": [
+            "onSave",
+            "onCreation"
```

Comment:
this option name seems to be a little bit confusing to me since I don't know what "creation" refers to here - if i understand correctly it's mote like `onTyping`